scripts/assets/commands/balena-idle

```diff
@@ -0,0 +1,40 @@
+#!/bin/bash
+
+set -e
+
+usage() {
+	cat <<EOUSAGE
+This script basically idles to keep the container running, while periodically printing to the logs that "Idling...".
```

Comment:
Rephrase as " A small script that runs an idle process to keep the container running, while periodically printing the log: "Idling...""

Reply:
@shaunmulligan, updated.